frontend/src/App.tsx

```diff
@@ -24,7 +24,9 @@ const App = () => {
               <Route exact path="/project/:id" render={({ match }) => <EditProject id={match.params.id} />} />
               <Route exact path="/projects" render={() => <ViewProject />} />
               <Route exact path="/error" render={() => <PageNotFound />} />
-              <Route exact path="/employee" render={() => <CreateEmployee />} />
+              <Route exact path="/businessAccount/new" render={() => <CreateBusinessAccount />} />
+              <Route exact path="/client/new" render={() => <CreateClientAccount />} />
```

Comment:
Change here to clientAccount instead of client

Reply:
done